openapi.yml

```
@@ -0,0 +1,302 @@
+      type: object
+      properties:
+        message:
+          type: string
+          description: Authentication error message
+          example: "Unauthenticated."
+
+tags:
+  - name: Authentication
+    description: User authentication endpoints (register, login, logout)
+  - name: User
+    description: User profile and information endpoints 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)